project/threads/templates/threads/static_templates/static_nav.html

```diff
@@ -11,20 +13,20 @@
 
         <!-- Right Nav -->
         <ul class="right hide-on-med-and-down">
-            <li><a class="dark-white-text menu-text" href="howitworks">how it works</a></li>
-            <li><a class="dark-white-text menu-text" href="about">about us</a></li>
-            <li><a class="dark-white-text menu-text" href="support_us">support us</a></li>
-            <li><a href="login" class="waves-effect waves-light btn login-btn">Login / Sign Up</a></li>
+            <li><a class="dark-white-text menu-text" href="howitworks">{% trans "how it works" %}</a></li>
+            <li><a class="dark-white-text menu-text" href="about">{% trans "about us" %}</a></li>
+            <li><a class="dark-white-text menu-text" href="support_us">{% trans "support us" %}</a></li>
+            <li><a href="login" class="waves-effect waves-light btn login-btn">{% trans "Login" %} / {% trans "Sign Up" %}</a></li>
```

Comment:
This may be a consideration for translators, so let's combine the strings
```suggestion
            <li><a href="login" class="waves-effect waves-light btn login-btn">{% trans "Login / Sign Up" %}</a></li>
```